src/hooks/api/classes/Message.ts

```
@@ -0,0 +1,45 @@
+  private client: WASocket;
+  private upserts: BaileysEventMap['messages.upsert'];
+
+  constructor(client: WASocket, upserts: BaileysEventMap['messages.upsert']) {
+    this.client = client;
+    this.upserts = upserts;
+  }
+
+  public async queueAllTasks() {
+    const semaphore = new Semaphore(5);
+    const taskPromises: Promise<void>[] = [];
+    let failedTasks: number = 0;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)